packages/google-cloud-tasks/package.json

```
@@ -75,7 +75,7 @@
     "compile:esm": "tsc -p ./tsconfig.esm.json && cp -r esm/src/json-helper.cjs build/esm/src/json-helper.cjs",
     "babel": "babel esm --out-dir build/cjs --ignore \"esm/**/*.d.ts\" --extensions \".ts\" --out-file-extension .cjs --copy-files",
     "compile:cjs": "tsc -p ./tsconfig.json && npm run babel",
-    "compile": "npm run compile:esm && npm run compile:cjs && cp -r protos build/protos"
+    "compile": "npm run compile:esm && npm run compile:cjs && rm -rf build/protos && cp -r protos build/protos"
```

To enable non-POSIX compatibility (e.g. Windows) we could make this an inline node script rather than `rm -rf`:
```suggestion
    "compile": "npm run compile:esm && npm run compile:cjs && node -e \"fs.rm('build/protos', {recursive: true})\" && cp -r protos build/protos"
```

---

Since technically we don't support Windows (we don't have windows tests), and *technically speaking* you could compile this library without having Node, I think it's better to keep as is.

---

With `npm` as a prerequisite means there is a version of `node` available in the `$PATH`. Also we do support Windows; while we don't have any OS-specific tests customers are certainly using Windows. Here are a few examples where this can bite customers:
- https://github.com/actions-on-google/smart-home-dashboard/issues/43

Either way, this would be best practice, but no one's filed an issue yet.